app/services/monitoring-stations/fetch-monitoring-stations.service.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+/**
+ * Fetches the matching monitoring station and associated licences needed for the view
+ * `/monitoring-stations/{monitoringStationId}/view` page
+ * @module FetchLicenceService
```

Comment:
This module name needs updating 
```suggestion
 * @module FetchMonitoringStationsService
```